com.github.ransome1.sleek.appdata.xml

```diff
@@ -8,6 +8,12 @@
   <metadata_license>MIT</metadata_license>
   <project_license>MIT</project_license>
 
+  <content_rating type="oars-1.0" />
+  <releases>
+    <release version="0.4.1" date="2020-12-13"/>
+  </releases>
+  <url type="github">https://github.com/ransome1/sleek</url>
```

Comment:
this as an invalid type.
See https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-url